solc/CommandLineInterface.cpp

```diff
@@ -604,6 +604,12 @@ bool CommandLineInterface::processInput()
 			 << boost::diagnostic_information(_exception);
 		return false;
 	}
+	catch (UnimplementedFeatureError const& _exception)
+	{
+		cerr << "Unimplemented compliler feature:" << endl
```

Comment:
Doh.

Happy to change them to "Unimplemented feature".  I think I was mostly keying off the language in the other messages.